profile/README.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="/profile/logo.png" height="30%" width="30%" />
+</p>
+
```

Comment:
All the following badges need to be centre aligned.

Reply:
Done